projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
+
+@Component({
+  selector: 'admin-library-detail-view',
+  templateUrl: './library-detail-view.component.html',
+  styles: []
+})
+export class LibraryDetailViewComponent implements DetailRecord, OnInit {
+
+  /** Observable resolving record data */
+  record$: Observable<any>;
+
+  /** Resource type */
+  type: string;
+
+  /** Record */
+  record: object;
+
+  constructor() { }
+
+  ngOnInit() {
+    this.record$.subscribe(record => {
+      this.record = record.metadata;
```

Comment:
Seems overkill. See my similar comment to your other PR.

Reply:
done